### PR TITLE
Update CHANGELOG.md for v1.1.5 release

### PR DESCRIPTION
**Pull request by phbnf**
### Checklist
- [x] I have updated the [CHANGELOG](CHANGELOG.md).
  - Adjust the draft version number according to [semantic versioning](https://semver.org/) rules.
- [ ] I have updated [documentation](docs/) accordingly.

### CODE REVIEW

**Comment by roger2hk** [comment]
You may want to release `google/trillian` first and then `google/certificate-transparency-go` after upgrading to the latest trillian version, so it would include everything new.

**Comment by phbnf** [comment]
I've cut the v1.1.5 release already. I can do the following, let me know if that's ok with you:
 - release google/trillian
 - upgrade google/certificate-transparency-go to use the last google/trillian release
 - then release google/certificate-transparency-go again

**Comment by roger2hk** [comment]
> I've cut the v1.1.5 release already. I can do the following, let me know if that's ok with you:
> 
> * release google/trillian
> * upgrade google/certificate-transparency-go to use the last google/trillian release
> * then release google/certificate-transparency-go again

That's a great idea. Thanks!

**Review by roger2hk — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## HEAD
 
+## v.1.1.5
```
[the remark this answers is not in the document]
Comment (roger2hk) [reply CHANGELOG.md:5]: @phbnf gentle ping

[the remark this answers is not in the document]
Comment (phbnf) [reply CHANGELOG.md]: Done, let's submit this one, and then cut a new release with the last Trillian release.

**Review by roger2hk — APPROVED** [review]

**Pull request merged**